src/Microsoft.Health.Dicom.Core/Exceptions/ExistingReindexException.cs

```diff
@@ -13,32 +13,19 @@ namespace Microsoft.Health.Dicom.Core.Exceptions;
 /// <summary>
 /// The exception that is thrown when a re-index request is submitted while one is already active.
 /// </summary>
-public sealed class ExistingReindexException : Exception
+public sealed class ExistingReindexException : ExistingOperationException
 {
-    /// <summary>
-    /// Gets the reference to the existing re-index operation.
-    /// </summary>
-    /// <value>The <see cref="OperationReference"/> for the existing operation, if specified.</value>
-    public OperationReference ExistingOperation { get; }
-
-    /// <summary>
-    /// Initializes a new instance of the <see cref="ExistingReindexException"/> class.
-    /// </summary>
-    public ExistingReindexException()
-    { }
-
     /// <summary>
     /// Initializes a new instance of the <see cref="ExistingReindexException"/> class.
     /// </summary>
     /// <param name="operation">The operation reference for the existing re-index operation.</param>
     /// <exception cref="ArgumentNullException"><paramref name="operation"/> is <see langword="null"/>.</exception>
     public ExistingReindexException(OperationReference operation)
         : base(
+            operation,
             string.Format(
                 CultureInfo.CurrentCulture,
                 DicomCoreResource.ExistingReindexOperation,
```

Comment:
Do we need unique messages, or can they combined since we know the operation name based on the exception

Reply:
I think it is better to give unique messages for the customer as below
`There is already an active update operation with ID '{0}'`
or
`There is already an active re-index operation with ID '{0}'`
We are not giving any of the exception details in the response.

Reply:
Let me know if you have a different thought.

Reply:
Couldn't we make the names `"update"` and `"re-index"` placeholders in the format string like `"There is already an active {0} operation with ID '{1}'"`